proto/LogProtocol.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+option optimize_for = SPEED;
+
+enum ColumnFlagType {
```

Comment:
We have more column flag now, ref: https://github.com/pingcap/ticdc/blob/e0dcfb707c531292111bf5a4d3a12e8d52e8c189/cdc/model/sink.go#L43-L58